internal/show/brokers.go

```diff
@@ -45,6 +46,7 @@ func Brokers(clusterInfo *cluster.Info, status reporter.Interface) bool {
 
 	brokers := brokerList.Items
 	if len(brokers) == 0 {
+		fmt.Println("   Broker not installed")
```

Comment:
I see your sample output in the commit message, and to me it looks like it would need an extra space to align. I wonder if we need the complexity of indentation at all? :shrug:

Reply:
Use the status to report the message.

```suggestion
		status.Success("No brokers found")
```

Reply:
"No brokers found" was my initial thought. But not installed sounds more accurate.

Reply:
A warning indicates it could be a problem but it's not required for a broker to be installed so this doesn't qualify as a warning.